databricks/koalas/indexes.py

```diff
@@ -587,6 +587,66 @@ def copy(self, name=None):
             result.name = name
         return result
 
+    def droplevel(self, level):
+        """
+        Return index with requested level(s) removed.
+        If resulting index has only 1 level left, the result will be
+        of Index type, not MultiIndex.
+
+        Parameters
+        ----------
+        level : int, str, or list-like, default 0
+            If a string is given, must be the name of a level
+            If list-like, elements must be names or indexes of levels.
+
+        Returns
+        -------
+        Index or MultiIndex
+
+
+        Examples
+        --------
+        >>> midx = ks.DataFrame({'a': ['a', 'b']}, index=[['a', 'x'], ['b', 'y'], [1, 2]]).index
+        >>> midx  # doctest: +SKIP
+        MultiIndex([('a', 'b', 1),
+                    ('x', 'y', 2),
+                   )
+
+        >>> midx.droplevel([0, 1])
+        Int64Index([1, 2], dtype='int64')
+
+        >>> midx.droplevel(0) # doctest: +SKIP
+        MultiIndex([('b', 1),
+                    ('y', 2)]
+                   )
+        """
+        if not isinstance(level, (tuple, list)):
+            level = [level]
+
+        for l in level:
```

Comment:
not to be necessarily fixed, just IMO,  it should better avoid `l` for naming as possible.

because It is often confused with the number `1` , and can impair readability. 😄